### PR TITLE
Add shrinkwrap to the makefile

### PR DESCRIPTION
**Pull request by Alduhoo**
PR for #76, making this my first contribution to an open source project. Thanks :)

### CODE REVIEW

**Comment by basicallydan** [comment]
@Alduhoo fantastic! Thanks for the contribution, o will merge as soon as my laptop has regained some battery charge :)

Welcome to OSS :tada:

File: makefile [hunk 1]
```diff
@@ -7,6 +7,7 @@ help:
 	@echo "  test        runs tests"
 	@echo "  compile     sets up your js files for production"
 	@echo "  serve       run the webserver"
+	@echo "  shrinkwarp  resets the shrinkwrap.json file"
```
Comment (basicallydan) [inline makefile]: @Alduhoo Small mistake, sorry: It says `shrinkwarp`, not `shrinkwrap` :D can you just fix this typo and I'll auto-merge?

**Comment by Alduhoo** [comment]
Fixed the typo, nice catch! :) Thanks!

**Comment by basicallydan** [comment]
@Alduhoo Great stuff - thank you!

**Pull request merged**